src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -110,7 +110,9 @@ function WorkspaceAccountingPage({policy}: WithPolicyProps) {
                 wrapperStyle: [styles.sectionMenuItemTopDescription],
                 shouldShowRightComponent: true,
                 title: translate('workspace.accounting.qbo'),
-                description: isSyncInProgress ? translate('workspace.accounting.connections.syncStageName', 'quickbooksOnlineImportCustomers') : translate('workspace.accounting.lastSync'),
+                description: isSyncInProgress
+                    ? translate('workspace.accounting.connections.syncStageName', {scope: 'quickbooksOnlineImportCustomers'})
```

Comment:
@brandonhenry This will cause the regression:
The output of this line will be `Translation missing for stage: [object Object]` , but the expected output should be`Importing customers`